erizo/src/erizo/rtp/BandwidthEstimationHandler.h

```diff
@@ -72,6 +76,12 @@ class BandwidthEstimationHandler: public Handler, public RemoteBitrateObserver,
   uint32_t bitrate_, last_send_bitrate_;
   uint64_t last_remb_time_;
   bool running_;
+  struct RRPackets {
```

Comment:
I'd prefer to have the definition at the beginning of the file (just before RemoteBitrateEstimatorPicker, for instance). And we could use `RRPacket audio_rr_, video_rr_;` here